### ISSUE TITLE
PLS Update map for ATS & ETS2

### ISSUE THREAD

**Issue by Sergeyz370** [SHWotever/SimHub#1621]
Describe the error
There is no state of Nebraska on the ATS map. Please update the map of America and ETC2. Thank you."

Simhub version
9.3.1

**Comment by DopeEx** [comment]
In ETS2 there were also some map updates that are not yet available, I would also be happy about an update. Thanks a lot.

**Comment by Sergeyz370** [comment]
Thank you very much!

**Comment by SHWotever** [comment]
Hi !

Both maps have been updated, make sure to clear the exsting maps to get them reloaded
![image](https://github.com/SHWotever/SimHub/assets/2207331/431c7df4-e851-4c42-908f-0ef4d691d5ff)

Nicolas

**Comment by Sergeyz370** [comment]
Yes, I tried, but some parts of the road are not displayed, including the new DLC, and leads parallel to the road.

**Comment by Sergeyz370** [comment]
Thank you very much for your efforts, everything has worked
